userspace/otpilot/src/main.rs

```diff
@@ -148,9 +154,19 @@ fn run() -> TockResult<()> {
 
     //////////////////////////////////////////////////////////////////////////////
 
+    // We assume that we've already done all boot-time checks at this point.
+
+    // Deassert BMC resets.
+    // TODO(osk): Do something with the result codes.
```

Comment:
"For now", maybe assert success?

Reply:
We need to implement proper "boot time checks" during which we'd want to handle errors like that better, e.g. by resetting Haven or so. I think of this section as a placeholder to implement something.